testdata/src/whitespace/whitespace_test.go

```diff
@@ -45,9 +45,15 @@ func TestWithNewlinesFlat(t *testing.T) {
 	fmt.Println("two-newlines\ntwo-newlines\ntwo-newlines")
 }
 
+func TestControl(t *testing.T) {
```

Comment:
I'd prefer to not mix this with the whitespace testdata. Can you move this into a new test package (called something like `illegalxmlchars`), with its own output and report testdata files?